draft-ietf-quic-version-negotiation.md

```diff
@@ -387,6 +387,40 @@ lifetime of the connection; endpoints MUST NOT change the version that they
 consider to be in use based on the Version field of long headers as that field
 could be forged by attackers.
 
+# Minimum support
+
+Hosts can support this standard incrementally.
+
+{{INV}} and {{QUIC}} already describe how servers send Version Negotiation
+packets in response to long header packets with unsupported version fields, even
+if they support only one version. Doing so prevents long timeouts at the client,
+but servers might not send the packet multiple times for the same connection, or
+when under stress.
+
+If a server supports two or more QUIC versions and sends Version Negotiation
+packets, it MUST send the Version Information Transport Parameter {{vers-info}},
```

Comment:
I feel like this just repeats the requirements from the downgrade section. Do we need it?